lib/convertor.js

```diff
@@ -21,6 +21,7 @@ const compOps = {
 const isOp = key => key.charAt(0) === '$';
 const isLogicOp = key => isOp(key) && _.includes(logicOps, key);
 const isCompOp = key => isOp(key) && _.includes(_.keys(compOps), key);
```

Comment:
Hating myself for breaking my own rule about shortening names here - something for later - `comparisonOp` FFS

Reply:
BADABOOOOM

`bdabo()`